jwql/website/apps/jwql/templates/base.html

```diff
@@ -115,6 +115,11 @@
 	        		<img src={{ static('img/STScIlogo_dark.png') }}   height=36>
 	        		<img src={{ static('img/JWSTlogo.png') }}   height=36>
 	        	</a>
+     	      	<div style="float:right; width:180px; text-align: center; color:white; font-size:12px">
```

Comment:
@bhilbert4 I think its good habit to use a `css` `class` or `id` here instead of defining a `style` in the `div` itself.  You could do something like:

```html
<div id='version-div'>
```

and then in `jwql.css`:

```css
#version-div {
    float: right;
    width: 180px;
    text-align: center;
    color: white;
    font-size: 12px;
}
```

This way, styles are more consolidated in `jwql.css` instead of scattered around the templates themselves.

Reply:
Thanks. I was intending to get to that after figuring out my problems with the version text, and then it slipped my mind. Fixed now.